migration_pipeline.py

```diff
@@ -6,15 +6,20 @@
 from checkpoint_service import CheckpointService
 
 
-def generate_session() -> str:
-    return datetime.now().strftime('%Y%m%d%H%M%S')
+def generate_session(args) -> str:
+    if args.validate_pipeline:
+        prefix = 'V'
+    else:
+        prefix = 'M'
+
+    return prefix + datetime.now().strftime('%Y%m%d%H%M%S')
```

Comment:
why use different sessions ?

Reply:
Session is just a unique id for pipeline runs. Adding prefix can let users distinguish different sessions easily. It's useful in the scenario of validation pipeline which will have 3 sessions:
1) Session A for export and import.
2) Session B for export destination.
3) Session C for validation.

Reply:
makes sense